packages/slate-hyperscript/src/index.js

```diff
@@ -77,6 +77,13 @@ function createHyperscript(options = {}) {
       .filter(child => Boolean(child))
       .reduce((memo, child) => memo.concat(child), [])
 
+    // by convention, strip attributes prefixed with double underscores
+    for (const key in attributes) {
+      if (key.lastIndexOf('__', 0) === 0) {
```

Comment:
This might be a dumb question, but could we just use `key.startsWith('__')` instead here for clarity? I think the only difference is that IE doesn't support it but I think it would get transpiled on build.

Reply:
Oh hmm, I guess it would be slightly different in case something has more underscores, or ends with underscores.

Reply:
Actually, I just wasn't sure what polyfills were in there or what browsers Slate targets, but if we're good to use `startsWith`, I prefer it too! It should work just fine here (that thing is ugly).

Reply:
The [browserlist has IE11](https://github.com/ianstormtaylor/slate/blob/master/browserslist#L5). So I think babel will handle it properly? 🤷‍♂️

Reply:
I'm not _totally_ sure it's not in the repo, but searching around now I'm not seeing babel-polyfill itself required, which I think is how babel does all polyfilling to keep it out of core. Pretty beefy to bring the whole thing in. It'd be possible probably to just bring the `startsWith` polyfill from core-js, but then it's still overkill for one call. :)

@ianstormtaylor Does Slate have any polyfills or existing pattern for it?

Reply:
@alanctkc Slate uses stuff that's fine for newest browsers and expects people to polyfill things like `startsWith` or `closest` I think.